devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -74,6 +80,46 @@ bool YarpRobotLoggerDevice::open(yarp::os::Searchable& config)
         return false;
     }
 
+    if (this->setupRobotCameraBridge(params->getGroup("RobotCameraBridge")))
+    {
+        // Currently the logger supports only rgb cameras
+        if (m_cameraBridge->getMetaData().bridgeOptions.isRGBCameraEnabled)
+            // || m_cameraBridge->getMetaData().bridgeOptions.isRGBDCameraEnabled)
+        {
+            std::vector<int> rgbFPS;
+            if (!params->getParameter("rgb_cameras_fps", rgbFPS))
+            {
+                return false;
+            }
+
+            const auto& rgbCameras = m_cameraBridge->getMetaData().sensorsList.rgbCamerasList;
+            if (rgbFPS.size() != rgbCameras.size())
+            {
+                return false;
+            }
+
+            for (unsigned int i = 0; i < rgbFPS.size(); i++)
+            {
+                if (rgbFPS[i] < 0)
+                {
```

Comment:
maybe we could add an error message with the camera name having invalid fps in config, so it will be easy to debug?

Reply:
Done here 7ce407e9b2ab7d8ef829395e217971c8f0e13077